temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -234,3 +234,84 @@ message GetRegionResponse {
     // The temporal cloud region.
     temporal.api.cloud.region.v1.Region region = 1;
 }
+
+message GetAPIKeysRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100.
+    int32 page_size = 1;
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
+    // Filter api keys by identity id - optional.
+    string identity_id = 3;
+    // Filter api keys by identity type - optional.
+    string identity_type = 4;
+}
+
+message GetAPIKeysResponse {
+    // The list of API keys in ascending ids order
+    repeated temporal.api.cloud.identity.v1.APIKey api_keys = 1;
```

Comment:
So in the list the user won't know the owner of each of the apikeys, right?
Maybe we should add identity type and if to the APIKey message.

Reply:
oh right let me add owner

Reply:
done